docs/general/manage-tokens/oasis-wallets/browser-extension.mdx

```diff
@@ -0,0 +1,266 @@
+
+:::caution
+
+Your mnemonic phrase is required to access your wallet. Be sure to **store it in
+a secure location**. If you lose or forget your mnemonic phrase, you will lose
+access to your wallet and any token funds contained in it.
+
+:::
+
+:::danger
+
+**Never share your mnemonic keyphrase**. Anyone with your keyphrase can access
+your wallet and your tokens.
+
+:::
+
+Now reenter the mnemonic words as a proof that you backed up the mnemonic and
+click on the "Confirm" button. Once done, you will see your balance on the
+**Oasis consensus layer**.
+
+![Account history](../../images/wallet/ext/account_history.png)
+
+## Import an Existing Account
+
+In the top-right corner, click your account icon to open the Account management
+menu.
+
+![Account Management - Importing Accounts](../../images/wallet/ext/accounts1.png)
+
+Click "Import" and select the:
+
+- "Oasis Private Key" to import your existing Oasis private key to be used by
+  Cipher ParaTime; or
+- "Ethereum-compatible Private Key" to import your existing Ethereum Account to
+  be used by the EVM-compatible Sapphire or Emerald.
+
+:::info
+
+We assume that you already have your public/private keypair. If you don’t have
+one yet, please go and create one. Store your Private Key, because you will
+need it in the Oasis Wallet Browser Extension.
+
+:::
+
+![Importing Ethereum-compatible Account with Private Key](../../images/wallet/ext/import.png)
+
+Fill in the "Account name" that will appear later in the Account Management screen.
+
+![Imported Account Name](../../images/wallet/ext/import2.png)
+
+Next, paste your Base64-encoded Oasis or hex-encoded Ethereum private key.
+
+:::note
+
+You will need to import **the private key and not the mnemonics**. You can
+derive a private key from the mnemonics with BIP39→BIP44 converter. For example
+by [importing](../cli/wallet.md#import) the mnemonic into the Oasis CLI and
+[exporting it](../cli/wallet.md#export).
+
+:::
+
+![Entering Ethereum address Private Key (BIP44)](../../images/wallet/ext/import3.png)
+
+Your newly imported account will, depending on the account type, appear under
+the "Oasis Account" or "Ethereum-compatible Account" sections in the Account
+Management screen. Check that the Oasis or Ethereum address shown at the bottom
+that it matches the address shown to you when you generated the keypair.
+
+:::danger
+
+In older versions of the Oasis Wallet Browser Extension there was also another
+bech32-encoded version of the Ethereum address shown in the Ethereum-compatible
+wallet. This address is used in the backend for setting the allowance policy
+on the Oasis network. **If you can see it, you should immediately update
+your Wallet Extension! The bech32-encoded address of the Ethereum-compatible
+account must never be used for transferring ROSE to. The signature schemes are
+incompatible (ECDSA versus ed25519) and those tokens would not be accessible
+anymore!**
```

Comment:
I think this should say "withdrawing to consensus layer". And for the signature schemes it should rather say that consesus layer only supports Ed25519.